### PR TITLE
Create Get-WhiteboardsForTenant.md

### PR DESCRIPTION
**Pull request by samanehrajabi**

### CODE REVIEW

**Comment by samanehrajabi** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — CHANGES_REQUESTED** [review]
Agree with @tseward suggestions

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+```powershell
+Get-WhiteboardsForTenant [-Geography] <String>
+  [-IncrementalRunName] <String>
+  [-ForceAuthPrompt]
+  [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Gets all the whiteboards in a tenant in a specified geography. Returns a list of whiteboard objects. The data returned is precalculated and therefore not realtime. Results are precalculated approximately every two weeks.
```
Comment (JohanFreelancer9) [inline whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md:30]:
```suggestion
Gets all the whiteboards in a tenant in a specified geography. Returns a list of whiteboard objects. The data is pre-calculated approximately every two weeks and is not real-time.
```

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+lastViewedTime:            lastViewedTime-value
+meetingId:                 meetingId-value
+eTag:                      eTag-value
+```
+
+## PARAMETERS
+
+### -Geography
+
+Required. The geography to look for board owners in. Accepted values are: Europe, Australia, or Worldwide (all boards not in Australia or Europe).
```
Comment (JohanFreelancer9) [inline whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md:68]:
```suggestion
The geography to look for board owners in. Accepted values are Europe, Australia, or Worldwide (all boards not in Australia or Europe).
```

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+Applicable: Microsoft Whiteboard
+```
+
+### -IncrementalRunName
+
+Optional. Saves incremental progess as the cmdlet runs. Use to resume a partially completed run. Use the same RunName on later calls to continue a previously cancelled/failed run. Writes progress and results to .txt files in the current directory:
```
Comment (JohanFreelancer9) [inline whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md:85]:
```suggestion
Saves incremental progress as the cmdlet runs. Use to resume a partially completed run. Use the same RunName on later calls to continue a previously canceled or failed run. Writes progress and results to .txt files in the current directory:
```

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+ - "Whiteboards-*.txt" contains the incremental results containing whiteboard objects for the tenant where * is the provided RunName. 
+ - "WhiteboardAdminRun-*.txt" contains the state of the current where * is the provided RunName. This file should not be modified manually
```
Comment (JohanFreelancer9) [inline whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md:87]:
```suggestion
 - "WhiteboardAdminRun-*.txt" contains the current state current where * is the provided RunName. This file should not be modified manually.
```

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+Position: 2
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+Applicable: Microsoft Whiteboard
+```
+
+### -ForceAuthPrompt
+
+Optional. Always prompt for auth. Use to ignore cached credentials.
```
Comment (JohanFreelancer9) [inline whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md:104]:
```suggestion
Always prompt for authentication. Use to ignore cached credentials.
```

**Comment by samanehrajabi** [comment]
Thanks @JohanFreelancer9 and  @tseward for your reviews. Applied changes based on your feedbacks.

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+### EXAMPLE 1
+
+```powershell
+PS C:\> Get-WhiteboardsForTenant -Geography Europe
+```
+
+Gets all the whiteboards associated with the caller's tenant in Europe as a list of whiteboard metadata objects.
+
+### Output
```
[the remark this answers is not in the document]
Comment (samanehrajabi) [reply whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md:42]: Changed to a real output with dummy values.

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
```
Comment (dariomws) [inline whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md:42]: Please check how to format the yaml code to include the output in the following link:
Output

**Review by dariomws — CHANGES_REQUESTED** [review]
Please check my comment.

This cmdlet will be included in the next module release?

Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

File: whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md (hunk 7 above)
Comment (samanehrajabi) [reply whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardsForTenant.md]: Thanks for reviewing @dariomws. Applied changes to example output based on the document. The cmdlet is already included, just haven't been documented.

**Comment by samanehrajabi** [comment]
@get-itips and @dariomws:
Thanks for reviewing. I have updated the document. Please let me know if there is anything else that needs to change.

**Review by dariomws — APPROVED** [review]

**Comment by dariomws** [comment]
@samanehrajabi, thanks!

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**